qiskit/providers/backend.py

```diff
@@ -229,3 +241,445 @@ class can handle either situation.
+                f"This backend's operations: {invalid_str} only apply to a subset of "
+                "qubits. Using this property to get 'basis_gates' for the "
+                "transpiler may potentially create invalid output"
+            )
+            logger.warning(msg)
+        return list(self.target.operation_names)
+
+    @property
+    @abstractmethod
+    def target(self):
+        """A :class:`qiskit.transpiler.Target` object for the backend.
+
+        :rtype: Target
+        """
+        pass
+
+    @property
+    def num_qubits(self) -> int:
+        """Return the number of qubits the backend has."""
+        return self.target.num_qubits
+
+    @property
+    def coupling_map(self):
+        """Return the :class:`~qiskit.transpiler.CouplingMap` object"""
+        return self.target.coupling_map()
+
+    @property
+    def instruction_durations(self):
+        """Return the :class:`~qiskit.transpiler.InstructionDurations` object."""
+        return self.target.durations()
+
+    @property
+    @abstractmethod
+    def max_circuits(self):
+        """The maximum number of circuits (or Pulse schedules) that can be
+        run in a single job.
+
+        If there is no limit this will return None
+        """
+        pass
+
+    @classmethod
+    @abstractmethod
+    def _default_options(cls):
+        """Return the default options
+
+        This method will return a :class:`qiskit.providers.Options`
+        subclass object that will be used for the default options. These
+        should be the default parameters to use for the options of the
+        backend.
+
+        Returns:
+            qiskit.providers.Options: A options object with
+                default values set
+        """
+        pass
+
+    def t1(self, qubits: Union[int, List[int]]) -> Union[float, np.array]:
+        """Return the T1 time of a given qubit
+
+        Args:
+            qubits: The qubit index to get the T1 time for. If
+                a list is specified the output will be a list with the
+                T1 time for the specified qubits in the same order.
+
+        Returns:
+            t1: the T1 time for the specified qubit(s) in seconds.
+
+        Raises:
+            NotImplementedError: if the backend doesn't support querying the
+                qubit properties for a qubit
+            ValueError: if T1 is not defined for the specified qubit
+        """
+        props = self.qubit_properties(qubits)
+        if isinstance(props, QubitProperties):
+            if props.t1 is None:
+                raise ValueError(f"T1 is not defined for qubit {qubits}")
+            return props.t1
+        out_list = []
+        for index, prop in enumerate(props):
+            if prop.t1 is None:
+                raise ValueError(f"T1 is not defined for qubit {qubits[index]}")
+            out_list.append(prop.t1)
+        return np.array(out_list)
+
+    def t2(self, qubits: Union[int, List[int]]) -> Union[float, np.array]:
```

Comment:
Same for this method, doesn't seem to be needed for base class.

Reply:
I'll drop this and T1 from the abstract backend. I just left it as a shortcut around `qubit_properties` to avoid a bit of boilerplate but I agree we don't really need it in the base class

Reply:
Removed in: https://github.com/Qiskit/qiskit-terra/pull/5885/commits/d2c279659e1be655388a45225f359c09b081f6ed